CONTRIBUTING.rst

```diff
@@ -38,6 +38,61 @@ General Guidelines
   from the cmd line to run the test suite).
 - Add yourself to doc/contributors.rst :)
 
+Authoring a Pull Request
+------------------------
+
+**Our Pull Request Policy is based on this** `Code Review Developer Guide <https://google.github.io/eng-practices/review>`_
+
+The expectation for any code author is to provide all the context needed in the space of a
+pull request for any engineer to feel equipped to review the code. Depending on the type of
+change, do your best to highlight important new functions or objects you’ve introduced in the
+code—think complex functions or new abstractions. Whilst it may seem like more work for you to
+adjust your pull request, the reality is your likelihood for getting review sooner shoots
+up.
+
+**Self Review Guidelines to follow**
+
+- If the PR is too large, split it if possible.
+    - Use 400 LoC (excluding test data and config changes) as a rule-of-thumb
```

Comment:
400 LoC is already a large PR excluding test data and config, what do we think about reducing this number?

Reply:
Yeah, that's valid. I will say 400 LoC is definitely the larger "upper-bound" in terms folks who institute "small bugfixes"; thoughts on 250?

@blink1073, @ShaneHarvey thoughts here?

Reply:
It isn't always within our control for some spec changes, but I think at least calling out a number here is good.  250 sounds like a fair goal.

Reply:
Switched to 250 :)